pyrate_limiter/sqlite_bucket.py

```diff
@@ -0,0 +1,84 @@
+import sqlite3
+from hashlib import sha1
+from os.path import join
+from tempfile import gettempdir
+from typing import List
+
+from pyrate_limiter.bucket import AbstractBucket
+
+
+class SQLiteBucket(AbstractBucket):
+    """Bucket backed by a SQLite database
+
+    Args:
+        maxsize: Maximum number of items in the bucket
+        identity: Bucket identity, used as the table name
+        path: Path to the SQLite database file; defaults to a temp file in the system temp directory
+        kwargs: Additional keyword arguments for :py:func:`sqlite3.connect`
+    """
+
+    def __init__(self, maxsize=0, identity: str = None, path: str = None, **kwargs):
+        super().__init__(maxsize=maxsize)
+        self._connection = None
+        self.connection_kwargs = kwargs
+        self.path = path or join(gettempdir(), "pyrate_limiter.sqlite")
```

Comment:
This kind of concern should almost always be the responsibility of applications, not libraries. I think you'd have a hard time finding any other library that uses SQLite that does things like this for the user, because the requirements are going to be very application-specific.

If you would like to make a follow-up PR to add some notes about this to the docs, I think that would be useful, though.